reportportal/templates/api-deployment.yaml

```diff
@@ -111,6 +111,14 @@ spec:
         - name: RP_BINARYSTORE_MINIO_SECRETKEY
           value: "{{ .Values.minio.secretkey }}"
        {{ end }}
+       {{ if .Values.minio.bucketPrefix}}
```

Comment:
better add one space at end of the variable before the curly brackets

Reply:
Fixed.